test/specs/prompts/input.js

```diff
@@ -53,4 +54,71 @@ describe("`input` prompt", function() {
     this.rl.emit("line", "");
   });
 
+  describe("given a terminal width", function() {
+
+    beforeEach(function() {
+      this.cliWidthStub = sinon.stub( utils, "cliWidth" );
+      this.cliWidthStub.returns(20);
+    });
+
+    afterEach(function() {
+      this.cliWidthStub.restore();
+    });
+
+    it("should clean short lines appropriately", function( done ) {
+      var prompt = new Input( this.fixture, this.rl );
+      var cleanSpy = sinon.spy( prompt, "clean" );
+      prompt.run(function() {
+        expect( cleanSpy.callCount ).to.equal(1);
+        expect( cleanSpy.calledWith(1) ).to.be.true;
+        done();
+      }.bind(this));
+
+      this.rl.line = "hello";
```

Comment:
That is correct as we use a mocked readline. So there's no usual Readline behavior during testing.